packages/language-server/src/server.ts

```diff
@@ -46,11 +53,22 @@ export function startLanguageServer(connection: vscode.Connection, env: RuntimeE
 	let hasConfigurationCapability = false;
 
 	connection.onInitialize((params: vscode.InitializeParams) => {
+		let isPnpInit = false;
```

Comment:
Not sure can we `require` this file multiple times for each workspace in context of a single server instance (will each import change previous behavior?)